applications/TuxedoDapp.md

```diff
@@ -6,160 +6,158 @@
 Provide feedback to users on transaction status and confirmations.
 
-**Transaction Confirmation**:
-DApp implements a transaction confirmation mechanism to ensure that users are informed about the progress and finalization of UTXO and NFT-based transactions such as Trade and Breed.
-
 **Personal Dashboard**:
+
 We will develop a personal dashboard for users to view and manage their NFT-based Kitties collections.
 Display detailed information about each NFT, including ownership history and transaction history.
 Manage NFT means, it includes below :
 
 1. Viewing NFTs:
+
    Users should be able to see a comprehensive list or visual representation of all the NFTs (kitties) they own.
-   Display detailed information about each NFT, including attributes such as below :
-   Name
-   parents(mom and dad)
-   free_breedings(numbers)
-   dna(H256 Hash value)
-   num_breedings(number)
-   Tradable(yes or No)
-   Price
-   Status (Ready for Raring, Tired (only for Dad), Had birth recently(only for Mom)).
-2. Trade :
-    Users should be able to buy the kitty from other users via the search kitty screen or directly from the Trade Kitty screen by inputting the kitty & owner details.
-3. Breed :
-    Users should be able to initiate breeding directly from the dashboard.
-4. Update :
-    Users should be able to update features such as Tradable status, price, and name from the dashboard by clicking on any cell which will navigate to update the kitty screen.
+   Display detailed information about each NFT, including attributes such as below:
+   - Name
+   - parents (mom and dad)
+   - free_breedings (numbers)
+   - DNA (H256 Hash value)
+   - num_breedings (number)
+   - Tradable (yes or No)
+   - Price
+   - Status (Ready for Raring, Tired (only for Dad), Had birth recently (only for Mom))
+
+2. Trade:
+
+    Users should be able to buy the kitty from other users via the search kitty screen or directly from the Trade Kitty screen by inputting the kitty & owner details
+
+3. Breed:
+
+    Users should be able to initiate breeding directly from the dashboard
+
+4. Update:
+
+    Users should be able to update features such as Tradable status, price, and name from the dashboard by clicking on any cell which will navigate to update the kitty screen
+
 5. Interactivity:
-   Make the dashboard interactive, allowing users to click on individual NFTs to access more detailed information or initiate specific actions.
+
+   Make the dashboard interactive, allowing users to click on individual NFTs to access more detailed information or initiate specific actions
 
 **Tech Stack:**
-- Polkadot-JS APIs for getting block data 
-- React Framework(React + Redux + React Router) OR Vue Framework (Vue.js with Vuex and the Vue Router)
-- TypeScript 
-- Babel for backward compatibility in older Browsers 
-- IndexedDB for Local database
-- IPFS for storing metadata related to the kitty such as Name, Trade status, Price, etc.
```

Comment:
@PieWol, Metadata of kitties is stored in the chain and it will not overload the chain. This was discussed with Joshy also and he agreed to our approach.